File: app/src/main/java/com/melikeey/ui/main/MainMvpPresenter.java
package com.melikeey.ui.main;

import com.melikeey.base.MvpPresenter;
import com.melikeey.di.PerActivity;

@PerActivity
public interface MainMvpPresenter<V extends MainMvpView> extends MvpPresenter<V> {

    void showInitialFragment();

}

File: app/src/main/java/com/melikeey/ui/main/MainMvpPresenter.kt
package com.melikeey.ui.main

import com.melikeey.base.MvpPresenter
import com.melikeey.di.PerActivity

@PerActivity
interface MainMvpPresenter<V : MainMvpView?> : MvpPresenter<V> {
    fun showInitialFragment()
}